Makefile

```diff
@@ -214,6 +224,7 @@ check: $(CARGO_OSDK)
 		(echo "Error: The combination of STD_CRATES and NOSTD_CRATES" \
 			"is not the same as all workspace members" && exit 1)
 	@rm /tmp/all_crates /tmp/combined_crates
+	@make typos
```

Comment:
I think the typo check should be moved to the last because typos are the least that we should worry. When our developers run `make check`, I think they should deal with important issues first and the lessor important ones later.